scripts/pipeline/orthology_benchmark.py

```diff
@@ -291,7 +320,10 @@ def calculate_goc_scores():
     parser.add_argument("--port", required=True, type=int, help="Database port")
     parser.add_argument("--user", required=True, type=str, help="Server username")
     parser.add_argument("--out_dir", required=True, type=str,
-                        help="Location for 'species_set/core_name.fasta' dumps")
+                        help="Location for 'species_set/' with 'core_name.fasta' dumps and "
+                             "corresponding GTF files")
+    parser.add_argument("--gtf_dir", required=True, type=str,
+                        help="Path to the directory containing subdirectories with GTF files.")
```

Comment:
It's good to be explicit about these GTF files being input files:
```suggestion
                        help="Path to the directory containing subdirectories with the ")
                             "GTF files that will be used as input."
```